src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -21,13 +21,13 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to the address book. "
             + "Parameters: "
             + PREFIX_TITLE + "TITLE "
-            + PREFIX_PHONE + "PHONE "
+            + PREFIX_DATE + "DATE "
             + PREFIX_DESCRIPTION + "DESCRIPTION "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_TITLE + "John Doe "
-            + PREFIX_PHONE + "98765432 "
+            + PREFIX_DATE + "01-01-2020-1200 "
```

Comment:
I personally feel that date and time should be separate, e.g. 01-01-2020 1200, we can discuss this in the team meeting tonight

Reply:
agree. and better put date as yyyy-mm-dd